### PR TITLE
Cargo "View Contents" Fix

### PR DESCRIPTION
**Pull request by Oyu07**
## What Does This PR Do

partially fixes https://github.com/ParadiseSS13/Paradise/issues/24556

Allows people to add "view content" as string

## Why It's Good For The Game

Crates that do not use contents, but rather spawn when broken does not list what will come in the box. This fixes it for secway, and allows players to manually add aditions

## Images of changes

image
image

## Testing

Order cargos
Look at contents
Checked normal cargo aswell to make sure I did not break them, I broke them once so be glad I tested this Grrrrrrrrrr

## Changelog
:cl:
fix: Secway Cargo order now displays contents
/:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
Change form tweak to fix on your changelog

File: code/modules/supply/supply_pack.dm [hunk 1]
```diff
@@ -44,18 +46,18 @@
...
-		for(var/path in contains)
-			if(!path)
-				continue
-			var/atom/movable/AM = path
-			manifest += "<li>[initial(AM.name)]</li>"
-			//Add the name to the UI manifest
-			ui_manifest += "[initial(AM.name)]"
-		manifest += "</ul>"
-	else
-		ui_manifest = list(manifest)
```
Comment (Oyu07) [inline code/modules/supply/supply_pack.dm:58]: This area is a revert from the emergency shuttle PR, testing the effects / changes currently but there should not be a difference

Comment (Oyu07) [inline code/modules/supply/supply_pack.dm:58]: Revert I guess? Testing if there has been any effects / changes currently

File: code/modules/supply/supply_pack.dm [hunk 2]
```diff
@@ -44,18 +46,18 @@
...
+		if(!path)
+			continue
+		var/atom/movable/AM = path
+		manifest += "<li>[initial(AM.name)]</li>"
+		//Add the name to the UI manifest
+		ui_manifest += "[initial(AM.name)]"
+	for(var/item in contains_special)
+		manifest += "<li>[item]</li>"
+		ui_manifest += "[item]"
+	manifest += "</ul>"
```
Comment (Oyu07) [inline code/modules/supply/supply_pack.dm:60]: here

**Review by Oyu07 — COMMENTED** [review]
Testing, will resolve when testing is done / no changes

Edit: seemed fine

**Review by S34NW — APPROVED** [review]

**Pull request merged**